src/visual.ts

```diff
@@ -511,15 +460,11 @@ module powerbi.extensibility.visual {
 
             this.dataView = options.dataViews[0];
 
-            if (options.type !== 4 && options.type !== 32 ) {
```

Comment:
I suppose that you shouldn't get rid of this condition. It skips the converter if the update type is resize.